WebView/app/build.gradle

```diff
@@ -53,7 +53,7 @@ dependencies {
     implementation "androidx.constraintlayout:constraintlayout:$constraintlayout_version"
     implementation "androidx.core:core-ktx:$core_ktx_version"
     implementation "androidx.fragment:fragment-ktx:$fragment_ktx_version"
-    implementation "androidx.webkit:webkit:$webkit_version"
+    implementation "androidx.webkit:webkit:1.3.0-beta01"
```

Comment:
Can you leave the `webkit_version` variable and change the version in the other build.gradle file?

Reply:
No we don't, all of the code that would require 1.3.0 was removed from this PR. It has been changed back to webkit_version.